File: cents.cpp
// Arup Guha
// Solution to 2005 UCF High School Contest Problem "Cents"
// 5/4/05

#include <iostream>
#include <fstream>

using namespace std;

void main() {

  // Open the input file.
  ifstream fin("cents.in");
  int numcases, dollars;

  // Read in the number of test cases.
  fin >> numcases;

  // Process each test case.
  for (int i=0; i<numcases; i++) {
    fin >> dollars;
    cout << 6*dollars << endl; // The tax in cents is 6 times the dollar
                               // amount of the purchase.
  }

  fin.close(); // close the input file.
}

File: cents.java
import java.io.*;

public class cents
{
	public static void main(String args) throws Exception
	{
		int i;
		String s;

		int NumberOfCases;
		int ValueInDollars;
				
		// Open file for reading
		BufferedReader fin = new BufferedReader(new FileReader("cents.in"));
		
		// Read the number of cases	
		s = fin.readLine();
		NumberOfCases = Integer.parseInt(s);

		// Loop through each case
		for(i = 0; i < NumberOfCases; i++)
		{
			// Read the number of dollars being charged
			s = fin.readLine();
			ValueInDollars = Integer.parseInt(s);

			// Output the sales tax in cents
			System.out.println(6*ValueInDollars);
		} 
	}
}
